python/pypto/language/parser/ast_parser.py

```
@@ -2647,6 +2691,84 @@ def _parse_scope_body(
+                if var is not None:
+                    external_vars.append(var)
+
+        # Annotated assignments with pl.Tensor type declare output variables
+        output_vars: list[ir.Var] = []
+        for node in stmt.body:
+            if isinstance(node, ast.AnnAssign) and isinstance(node.target, ast.Name):
+                var_name = node.target.id
+                try:
+                    var_type, _direction = self.type_resolver.resolve_param_type(node.annotation)
+                except Exception:
+                    continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)